classes/mender-update.bbclass

```diff
@@ -0,0 +1,2 @@
+
+
```

Comment:
This is left for reference. I am thinking about having different images for first install and then for doing update.